.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,65 @@
+name: PR actions
+'on':
+  pull_request:
+    branches:
+      - master
+jobs:
+  danger-ci:
+    name: Danger CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+        env:
+          RUNNER_TEMP: /tmp
+      - name: Danger CI
+        uses: vtex/danger@master
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+          REQUIRE_CHANGELOG_VERSION: false
```

Comment:
I think they need this to be true?

Reply:
I'm not familiar with the release process now

Reply:
See comment below

Reply:
Just talked to @emersonlaurentino. Only the core team is allowed to release a version, so this check will never be useful here :(